_schemaV2.graphql

```diff
@@ -5922,6 +5929,26 @@ type OrderedSetEdge {
 
 union OrderedSetItem = Artist | Artwork | FeaturedLink | Gene
 
+# A connection to a list of items.
+type OrderedSetItemConnection {
+  # Information to aid in pagination.
+  pageInfo: PageInfo!
+
+  # A list of edges.
+  edges: [OrderedSetItemEdge]
```

Comment:
Realistically all the Relay connections should be non-nullable, yeah? I think we don't do this anywhere but it should be safe and avoid all of the inevitable edges?...node?.id stuff that's going to happen when we move to strict null checks.

Reply:
There's a bigger discussion on that here: https://github.com/artsy/metaphysics/pull/2346 !

Reply:
Ah, exciting PR, that.